src/Raven.Server/Utils/Cli/CommandLineSwitches.cs

```diff
@@ -100,6 +104,11 @@ public static string[] Process(string[] args)
                 "-l | --log-to-console",
                 "Print logs to console (when run in non-interactive mode)",
                 CommandOptionType.NoValue);
+            _nonConsoleOption = _app.Option(
+                "--non-console",
```

Comment:
--no-color ?

Reply:
The idea was to write this flag when we are using a program without any console to write too,
so non-console will be better

Reply:
Still, it would be better to go with `--no-color` because this is what it does actually. It can be used outside of embedded.